src/main/java/software/bernie/techarium/block/voltaicpile/VoltaicPileBlock.java

```diff
@@ -0,0 +1,58 @@
+package software.bernie.techarium.block.voltaicpile;
+
+import net.minecraft.block.AbstractBlock;
+import net.minecraft.block.BlockRenderType;
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.fluid.FluidState;
+import net.minecraft.inventory.InventoryHelper;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IWorld;
+import net.minecraft.world.World;
+import software.bernie.geckolib3.core.IAnimatable;
+import software.bernie.geckolib3.core.PlayState;
+import software.bernie.geckolib3.core.builder.AnimationBuilder;
+import software.bernie.geckolib3.core.controller.AnimationController;
+import software.bernie.geckolib3.core.event.predicate.AnimationEvent;
+import software.bernie.geckolib3.core.manager.AnimationData;
+import software.bernie.geckolib3.core.manager.AnimationFactory;
+import software.bernie.techarium.block.base.TechariumBlock;
+import software.bernie.techarium.registry.ItemRegistry;
+import software.bernie.techarium.tile.voltaicpile.VoltaicPileTile;
+import software.bernie.techarium.trait.block.BlockBehaviours;
+import software.bernie.techarium.util.TechariumMaterial;
+
+public class VoltaicPileBlock extends TechariumBlock<VoltaicPileTile> {
+    public VoltaicPileBlock() {
+        super(BlockBehaviours.voltaicPile, AbstractBlock.Properties.of(TechariumMaterial.METAL));
```

Comment:
add side config to enable energy output in the wanted directions
```suggestion
        super(BlockBehaviours.voltaicPile, AbstractBlock.Properties.of(TechariumMaterial.METAL));
        for (Side side: Side.values()) {
            getFaceConfigs().put(side, FaceConfig.PUSH_ONLY);
        }
```